### DISCUSSION TITLE
AppCrashes with error "xref_count == xref_index"

### DISCUSSION THREAD

**Discussion by cgomezm** [dotnet/maui#23826] [Q&A]
Hey, 

Our application suddenly started crashing with error " * Assertion at /__w/1/s/src/mono/mono/metadata/sgen-tarjan-bridge.c:1174, condition `xref_count == xref_index' not met, function:processing_build_callback_data" 

There is no exception thrown at all and the app just closes. The only error message is this and we captured it from logcat
![image](https://github.com/user-attachments/assets/63465a3a-1de0-4bcc-9c7e-40a8382ee341)

**Comment by Syed-RI** [comment]
https://github.com/dotnet/android/issues/9377, https://github.com/dotnet/runtime/issues/106410 not a MAUI issue alber it experienced in MAUI. Not sure whats the solution here.
